pkgs/tools/virtualization/shipyard/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "shipyard";
+  version = "0.0.37";
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "shipyard-run";
+    repo = pname;
+    sha256 = "035a12sl1my1fgvjab9la6j0r79ph9kkrycqrzgg08iz6kn23pjz";
+  };
+
+  subPackages = [ "." ];
+  vendorSha256 = "03qhvrpm6d1h6501885zac2gxa9q449rs7y82r2arhz6mky4p2lw";
```

Comment:
Please move this line to be right below `src`.

Reply:
Done.